dpgen2/entrypoint/submit.py

```diff
@@ -236,7 +238,7 @@ def make_naive_exploration_scheduler(
                 sys_idx = jj.pop('conf_idx')
             conf_list = []        
             for ii in sys_idx:
-                conf_list += make_conf_list(sys_configs[ii], type_map)
+                conf_list.extend(sys_configs_lmp[ii])
```

Comment:
I do not see why the necessity of this change, it seems that the old and new code have almost the same behavior. Am I right?

Reply:
@wanghan-iapcm  Yes, they are almost the same, except that the new code is slightly faster when `conf_list`  is very large.

Reply:
@wanghan-iapcm I check it again and it turns out that I made a mistake. The code are almost the same as well as their performance even when `conf_list` is very large.